src/actions/recomputeReduxState.js

```diff
@@ -92,13 +92,29 @@ const modifyStateViaURLQuery = (state, query) => {
   if (query.tl) {
     state["tipLabelKey"] = query.tl===strainSymbolUrlString ? strainSymbol : query.tl;
   }
+  
   if (query.dmin) {
-    state["dateMin"] = query.dmin;
-    state["dateMinNumeric"] = calendarToNumeric(query.dmin);
+    const dateNum = calendarToNumeric(query.dmin);
+    if (dateNum===undefined || dateNum < state.absoluteDateMinNumeric || dateNum > state.absoluteDateMaxNumeric) {
+      console.error(`Requested dmin (min date) "${query.dmin}" is invalid`);
```

Comment:
I was confused why I kept seeing this error in the testing dataset when trying to view https://nextstrain-s-nextstrain-waprls.herokuapp.com/staging/rabies/YYYY-MM-DD-coloring?dmin=2020-01-01

It took me too long to realized this is a div only tree, so dmin/dmax/animate would not apply 🤦‍♀️ 
It would have been helpful to see a message that these queries do not work on a `divOnly` tree.

Reply:
I think there are two cases of invalid URL queries - when the UI has put us in an invalid state (bad!) and when we've added an invalid query ourselves. This falls in the latter group and in general I don't think we need to do too much here, but in this case it's simple enough that I added it:
```
console.error(`URL query "dmin=${query.dmin}" is invalid ${state.branchLengthsToDisplay==='divOnly'?'(the tree is not a timetree)':''}`);

```

(There are plenty of weird situations similar to this -- e.g. ask for the frequencies panel in a dataset without one:  [zika?d=frequencies](https://nextstrain.org/zika?d=frequencies).)

Reply:
Yup, just the additional hint that the tree is not a timetree is enough for me! These types of hints will be especially useful when we [surface console errors in the UI](https://github.com/nextstrain/auspice/issues/1813)